### DISCUSSION TITLE
Allow exporting selected items from layout

### DISCUSSION THREAD

**Discussion by rijkvanzanten** [directus/directus#12967] [Feature Requests]
Some layouts allow you to select a subset of items. It would be nice if you could then use the export flow to only export the selected items.

Couple of immediate questions:

- How does one indicate to only export the selected items? A new button?
- How do layouts tell the parent container to show/hide the "export selected" flow?
- Does the user still need to configure the regular export settings? I think _yes_ for fields / format, but _no_ for limit / filter as you have selected items

**Comment by lucasrubini** [comment]
I don't know if here is the right place to say that, but it would be really useful if we could export the selected items to PDF by using a predefined template (.liquid?).

If not, what would you recommend?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
